src/mds/Server.cc

```diff
@@ -5794,18 +5794,22 @@ int Server::parse_quota_vxattr(string name, string value, quota_info_t *quota)
       }
     } else if (name == "quota.max_bytes") {
       /*
-       * The "quota.max_bytes" must be aligned to 4MB if greater than or
-       * equal to 4MB, otherwise must be aligned to 4KB.
+       * The "quota.max_bytes" must be aligned to 4MiB if greater than or
+       * equal to 4MiB, otherwise must be aligned to 4KiB.
        */
       string cast_err;
       int64_t q = strict_iec_cast<int64_t>(value, &cast_err);
-      if(!cast_err.empty() ||
-         (!IS_ALIGNED(q, CEPH_4M_BLOCK_SIZE) &&
-          (q < CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4K_BLOCK_SIZE)))) {
-        dout(10) << __func__ << ":  failed to parse quota.max_bytes: "
+      if(!cast_err.empty()) {
+        dout(10) << __func__ << ": failed to parse quota.max_bytes: "
```

Comment:
I think I had added this but this looks quite redundant, it would print 
```
parse_quota_vxattr: failed to parse quota.max_bytes: <err>
```

Instead
```
failed to parse quota.max_bytes: <err>
```
looks better, what do you think @lxbsz?

Reply:
Do you mean remove the `__func__` ?

Reply:
If so I think we should keep it as from many other debug logs, which all have this, in this file.

Reply:
I see, `__func__` is used all over the place in Server code.